packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -149,6 +150,92 @@ export class ProductCard extends PureComponent {
         );
     }
 
+    renderBundlePriceBadge(bounds) {
+        const {
+            product: { type_id, price_view }
+        } = this.props;
+
+        if (type_id !== BUNDLE) {
+            return null;
+        }
+
+        // eslint-disable-next-line
+        let bundleBadgeLabel;
+
+        if (bounds === 'bottom') {
+            bundleBadgeLabel = 'From';
+
+            if (price_view !== 'PRICE_RANGE') {
+                bundleBadgeLabel = 'As low as';
+            }
+        }
+
+        if (bounds === 'top') {
+            bundleBadgeLabel = 'To';
+        }
+
+        return (
+            <p
+              mix={ {
+                  block: 'ProductCard',
+                  elem: 'PriceBadge'
+              } }
+            >
+                { __(bundleBadgeLabel) }
+            </p>
+        );
+    }
+
+    renderOldMaxBundlePrice() {
+        const {
+            product: { price_range }
+        } = this.props;
+
+        const oldMaxBundlePrice = `${ price_range.maximum_price.regular_price.value }`;
+        const finalMaxBundlePrice = `${ price_range.maximum_price.final_price.value }`;
+
+        if (finalMaxBundlePrice === oldMaxBundlePrice) {
+            return null;
+        }
+
+        return (
+            <del
+              block="ProductPrice"
+              elem="HighPrice"
+              aria-label={ __('Old product price') }
+            >
+                { roundPrice(`${ oldMaxBundlePrice }`) }
+            </del>
+        );
+    }
+
+    renderMaxBundlePrice() {
+        const {
+            product: { price_range }
+        } = this.props;
+
+        const oldMaxBundlePrice = `${ price_range.maximum_price.regular_price.value }`;
+        const finalMaxBundlePrice = `${ price_range.maximum_price.final_price.value }`;
+
+        return (
+            <p
+              aria-label={ __(`'Maximal product price: ${ finalMaxBundlePrice }'`) }
```

Comment:
As i mentioned before, translatable strings should be constant. But there is a way to put variables here.
Please use the following syntax:
```javascript
__(`Maximal product price: %s`, finalMaxBundlePrice)
```